sunpy/map/mapbase.py

```diff
@@ -1963,50 +1963,122 @@ def draw_grid(self, axes=None, grid_spacing: u.deg = 15*u.deg, annotate=True, **
                                                            rsun=self.rsun_meters,
                                                            **kwargs)
 
-    def draw_limb(self, axes=None, **kwargs):
+    def draw_limb(self, axes=None, *, resolution=1000, **kwargs):
         """
-        Draws a circle representing the solar limb
+        Draws the solar limb as seen by the map's observer.
+
+        The limb is a circle for only the simplest plots.  If the coordinate frame of
+        the limb is different from the coordinate frame of the plot axes, not only
+        may the limb not be a true circle, a portion of the limb may be hidden from
+        the observer.  In that case, the circle is divided into visible and hidden
+        segments, represented by solid and dotted lines, respectively.
 
         Parameters
         ----------
-        axes: `~matplotlib.axes` or None
-            Axes to plot limb on or None to use current axes.
+        axes : `~matplotlib.axes` or ``None``
+            Axes to plot limb on or ``None`` to use current axes.
+        resolution : `int`
+            The number of points to use to represent the limb.
```

Comment:
```suggestion
            The number of segments to use to represent the limb.
```
I think, because you use `resolution+1` in `linspace` below.

Reply:
It's actually both points and segments.  (There's a `[:-1]` in the later line.)  Because I use the word "segments" earlier in the docstring, I'd rather say "points" here.

Reply:
Ah I forgot for a closed path `npoints == nsegments`.